python_modules/dagster/dagster/_cli/project.py

```diff
@@ -32,6 +32,8 @@ def project_cli():
     "This CLI enables you to quickly bootstrap your project with an officially maintained example."
```

Comment:
Outside this review but would be useful to add "See 'list-examples'" or something to help users playing around with the CLI.